filter-map-view (#1054)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/14-noc-ui.md

# Overview

> Palette provides intuitive UI-based location monitoring for the clusters running at multiple locations. Palette displays the region set during the cluster creation process for public cloud clusters and the location on the UI Map. The user can set the location for private cloud clusters through the Palette UI. The user can monitor the location details of all the clusters running under a specific scope. 

Reviewer: ```suggestion
Palette provides an intuitive user interface (UI) based on location monitoring for the clusters running at multiple locations. The Palette 
UI displays the region set during the cluster creation process for public cloud clusters and the location on the UI map. You can set the location for private cloud clusters through the Palette UI. You can also monitor the location details of all the clusters running under a specific scope. 
```

# Overview

> Palette provides Intuitive UI-based location monitoring for the clusters running at multiple locations. For public cloud clusters Palette displays the `Region` set during the cluster creation process and displays the location on the UI Map. For private cloud clusters the user can set the location through the Palette UI. The user can monitor the location details of all the clusters running under a specific scope. 

Reviewer: ```suggestion
Palette provides intuitive UI-based location monitoring for the clusters running at multiple locations. Palette displays the region set during the cluster creation process for public cloud clusters and the location on the UI Map. The user can set the location for private cloud clusters through the Palette UI. The user can monitor the location details of all the clusters running under a specific scope. 
```
Minor cleanup

> * Login to the Palette console.

Reviewer: ```suggestion
* Login to [Palette](https://console.spectrocloud.com).
```

> * Select the cluster to which the location needs to be updated and go to the **cluster details** page of the cluster.

Reviewer: ```suggestion
* Select the cluster to which the location needs to be updated and go to the **Cluster details** page of the cluster.
```

> * Open **Settings** and then **Cluster Settings**.

Reviewer: ```suggestion
* Open **Settings** and then navigate to the **Cluster Settings**.
```

> * Select **Location** from the left menu and set the location of that cluster to the wizard and save the changes.

Reviewer: ```suggestion
* Select **Location** from the left menu, set the cluster's location, and save the changes.
```
"To that wizard" sounds awkward because although we are using a wizard, it's not what we save the changes to. The wizard is just a UI tool to help the user.

<br />

> * Log in to the Palette console and select **Clusters** from the left main menu.

Reviewer: ```suggestion
* Log in to Palette and select **Clusters** from the left **Main Menu**.
```

> * Go to **Map View Icon** below the `Clusters` tab.

Reviewer: ```suggestion
* Go to **Map View Icon** below the **Clusters** tab.
```
Clusters is a UI element so it needs bold.

The map will display all the cluster locations under that user’s scope. 

# Map Filters

> Palette Map Filters filter out specific cluster(s) using built-in or custom filters for an enhanced user experience. This map filter provides an enhanced user experience when you have many clusters dispersed geographically across multiple scopes and clouds. You have two types of filters:- **Built in Filters** and **Custom Filters**.

Reviewer: ```suggestion
Palette Map Filters filter out specific clusters using built-in or custom filters for an enhanced user experience. The map filter allows you to narrow down clusters that may be dispersed geographically, across multiple scopes, or different cloud providers. You have two types of filters:- **Built-in Filters** and **Custom Filters**.
```

Vale is correct in this feedback, we should not use plurals in parentheses.

> ## Built-in filters

Reviewer: ```suggestion
## Built-In Filters
```
Style guide ^

> The filters are available in the Palette console by default and can be selected from the **Add Filter** drop-down menu. You can have the following built-in filters from the Palette console.

Reviewer: ```suggestion
Built-in filters are available in the Palette console by default and can be selected from the **Add Filter** drop-down menu. You can use the following built-in filters.
```
This re-writes the sentence by specifying in the beginning what is available. The second sentence is re-written to active voice.

|**Built-In Filters** |
|---------------------|
|Deleted Only|
|Imported Only|

> |Updates Pending|

Reviewer: We should explain what each filter is for. Let's add a description column.

## Custom Filters

Palette supports a wide range of custom filters in a fixed format. To add a custom filter:
<br />

> * Log in to the Palette console and select **Clusters** from the left main menu.

Reviewer: ```suggestion
* Log in to Palette and select **Clusters** from the left **Main Menu**.
```

* Click on the **+Add Filter** button on the top menu and select **+ Add custom filter** from the top menu.

* The format for adding a cluster is as follows:

  `Conjunction - Condition - Operator - Value`

> * You can add more than one custom filter simultaneously, and they work together with the `conjunction` added.

Reviewer: ```suggestion
* You can add more than one custom filter simultaneously, and they work together with the chosen conjunction.
```

<br />

> <InfoBox>

Reviewer: This does not need an infobox. This can simply be part of the information before the table.

|and/or|Name|[operator](/clusters/cluster-management/noc-ui#operators) | Custom value|
|and/or|Profiles|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value |
|and/or|Status|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Environment|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Environment|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Health Status|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Deleted|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Read Only Import| [operator](/clusters/cluster-management/noc-ui#operators)|Custom value|
|and/or|Imported|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Updates Pending|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Tags|[operator](/clusters/cluster-management/noc-ui#operators) |Custom value|
|and/or|Region| [operator](/clusters/cluster-management/noc-ui#operators)|Custom value|

## Operators

> | |
> |-|

Reviewer: ```suggestion
```
We shouldn't use a markdown table for styling purposes.
